tests/javaRosa.js

```diff
@@ -783,7 +785,6 @@ define([
     describe("The javaRosaplugin with one language", function() {
         before(function(done) {
             util.init({
-                features: {rich_text: false},
```

Comment:
This felt safe since it only affected one other test, and that test only needed a tiny change to its XML and otherwise was not affected adversely.